File: tests/fixtures/goal_verify_v4/main/fix-nextjs-normalization/before/repro.py
#!/usr/bin/env python3
import sys
from lib import normalize

CASES = {
    str(index): (f"　{chr(0xFF10 + index)}{chr(0xFF10 + index)}　", str(index) * 2)
    for index in range(1, 10)
}
CASES["10"] = ("　１０　", "10")


def main(argv):
    if len(argv) != 2 or argv[1] not in CASES:
        return 2
    source, expected = CASES[argv[1]]
    actual = normalize(source)
    if actual != expected:
        print(f"expected {expected!r}, got {actual!r}", file=sys.stderr)
        return 1
    return 0


if __name__ == "__main__":
    raise SystemExit(main(sys.argv))

File: tests/fixtures/goal_verify_v4/a15/fix-nextjs-route-label/after/scripts/repro.mjs
#!/usr/bin/env node
import { readFile } from "node:fs/promises";
import { formatTask } from "../lib/label.mjs";

if (process.argv.length !== 3 || !process.argv[2].startsWith("fixture/task-")) {
  process.exit(2);
}
const fixture = JSON.parse(await readFile(process.argv[2], "utf8"));
const actual = formatTask(fixture.task);
if (actual !== fixture.expected) {
  console.error(`expected ${fixture.expected}, got ${actual}`);
  process.exit(1);
}
